project/plugins.sbt

```diff
@@ -19,6 +19,23 @@ addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.6.0")
 
 addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2") // formerly known as com.jsuereth:sbt-pgp
 
-addSbtPlugin("com.github.sbt" % "sbt-jacoco" % "3.4.0")
+// sbt-jacoco - workaround related dependencies required to download
+lazy val ow2Version = "9.5"
+lazy val jacocoVersion = "0.8.9-absa.1"
+
+def jacocoUrl(artifactName: String): String = s"https://github.com/AbsaOSS/jacoco/releases/download/$jacocoVersion/org.jacoco.$artifactName-$jacocoVersion.jar"
+def ow2Url(artifactName: String): String = s"https://repo1.maven.org/maven2/org/ow2/asm/$artifactName/$ow2Version/$artifactName-$ow2Version.jar"
+
+addSbtPlugin("com.jsuereth" %% "scala-arm" % "2.0" from "https://repo1.maven.org/maven2/com/jsuereth/scala-arm_2.11/2.0/scala-arm_2.11-2.0.jar")
+addSbtPlugin("com.jsuereth" %% "scala-arm" % "2.0" from "https://repo1.maven.org/maven2/com/jsuereth/scala-arm_2.12/2.0/scala-arm_2.12-2.0.jar")
```

Comment:
Isn't this a problem - adding both plugins for Scala 2.11 and for 2.12 regardless of which is used?

Reply:
It looks but it works.